src/main/resources/application.yaml

```diff
@@ -14,13 +14,39 @@ management:
 
 #If you use a database then uncomment below lines and update db properties accordingly
 spring:
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    show-sql: false
+    properties:
+      hibernate:
+        default_schema: dbcommondata
+        jdbc:
+          lob:
+            non_contextual_creation: true
+          time_zone: UTC
+    cache:
+      cache-names: token
+      caffeine:
+        spec: maximumSize=50,expireAfterWrite=1m
   flyway:
     enabled: true
     locations: ${COMMONDATA_DB:classpath:db/migration}
   config:
     import: "optional:configtree:/mnt/secrets/rd/"
+  main:
+    allow-bean-definition-overriding: true
   application:
-    name: Common Data Api
+    name: RD Commond Data API
+  security:
+      oauth2:
+        client:
+          provider:
+            oidc:
+              issuer-uri: ${OPEN_ID_API_BASE_URI:https://idam-web-public.aat.platform.hmcts.net/o}
+          registration:
+            oidc:
+              client-id: microservicekey-rd-commondata-api
+              client-secret: ${COMMON_DATA_API_S2S_SECRET:}
```

Comment:
Should not this be `COMMONDATA_API_S2S_SECRET`?

Reply:
Thanks @SujeethKumarAri , Good Catch.
Fixed it.